docs/react-native/v2/features/change-role.mdx

```diff
@@ -55,13 +104,67 @@ const onRoleChangeRequest = ({ requestedBy, suggestedRole }) => {
 hmsInstance.addEventListener(HMSUpdateListenerActions.ON_ROLE_CHANGE_REQUEST, onRoleChangeRequest);
 ```
 
-## Accept Request Role Change
+At this point, the app should show a prompt to the user asking for permission to accept or deny role change. 
 
-Once the peer with adequate permissions calls change role, you can either accept or reject the request.
+
+If the user wants to accept the request, the app should invoke `acceptRoleChangeRequest` on `HMSSDK` instance:
 
 ```js
 // call this if you accept role change else close the prompt
 await hmsInstance.acceptRoleChange()
     .then(d => console.log('Accept change role Success: ', d))
     .catch(e => console.log('Accept change role Error: ', e));
 ```
+
+
+Now, all peers in the room will receive an `HMSPeerUpdate.ROLE_CHANGED` callback so that they both can do the necessary UI updates. Now the user becomes a speaker and the audio publishing will start automatically.
```

Comment:
```suggestion
Now, all peers in the room will receive an `HMSPeerUpdate.ROLE_CHANGED` callback so that they all can do the necessary UI updates. Now the user becomes a speaker and the audio publishing will start automatically.
```